baselines/dom.es6.generated.d.ts

```diff
@@ -0,0 +1,104 @@
+
+/////////////////////////////
+/// DOM ES6 APIs
+/////////////////////////////
+
+interface AudioTrackList {
+    [Symbol.iterator](): IterableIterator<AudioTrack>
+}
+
+interface CSSRuleList {
+    [Symbol.iterator](): IterableIterator<CSSRule>
+}
+
+interface CSSStyleDeclaration {
+    [Symbol.iterator](): IterableIterator<string>
+}
+
+interface ClientRectList {
+    [Symbol.iterator](): IterableIterator<ClientRect>
+}
+
+interface DOMStringList {
```

Comment:
It meets the standard on WHATWG spec:

```WebIDL
interface DOMStringList {
  readonly attribute unsigned long length;
  getter DOMString? item(unsigned long index);
  boolean contains(DOMString string);
};
```

It has 1. an indexed property getter and 2. an integer-typed attribute named “length”.